test/test_nn.py

```diff
@@ -377,6 +377,7 @@ class NewCriterionTest(InputVariableMixin, CriterionTest):
     def __init__(self, *args, **kwargs):
         super(NewCriterionTest, self).__init__(*args, **kwargs)
         self.check_gradgrad = kwargs.get('check_gradgrad', True)
+        self.check_grad_target = kwargs.get('check_grad_target', False)
```

Comment:
I'm not sure we want to add more functionality to the CriterionTest because we're moving away from a notion of a "loss function" -- all losses should be modules. 

You should be able to write a [module test](https://github.com/pytorch/pytorch/blob/master/test/test_nn.py#L6025-L6035) for this. All module tests test grad of their inputs (and gradgrad), so you should be able to send in a target with requires_grad=True and see if it works.

Reply:
I see, I will add a new test for kl_div target as `module test`. Is there a way to pytest all/part of NewModuleTest only?

Reply:
Like test only specific tests? You can do `pytest test/test_nn.py -k "kl_div" -v` (if you have pytest installed) to test all tests in test_nn.py that have the string "kl_div" in them.

Reply:
Right, that's exactly what I wanted! But looks like pytest only runs on methods with prefix `test_`. Currently I have the following results if I do `pytest test/test_nn.py -k "kl_div" -v` or `pytest test/test_nn.py -k "kldiv" -v`

```
pytest test/test_nn.py -k "kl_div" -v
================================================================= test session starts ==================================================================
platform linux2 -- Python 2.7.14, pytest-3.3.2, py-1.5.2, pluggy-0.6.0 -- /home/weiyang/anaconda2/bin/python
cachedir: .cache
rootdir: /home/weiyang/pytorch, inifile:
collected 1128 items

=================================================================== warnings summary ===================================================================
test/test_nn.py
  cannot collect 'test' because it is not a function.

test/test_nn.py::TestBase
  cannot collect test class 'TestBase' because it has a __init__ constructor

-- Docs: http://doc.pytest.org/en/latest/warnings.html
```

Reply:
Yikes, pytest is case sensitive, try: `pytest test/test_nn.py -k "KLDiv"  -v`

Reply:
Bingo! It works like a charm! Thanks a ton!